ui/lib/pki/addon/components/parsed-certificate-info-rows.js

```diff
@@ -38,4 +38,11 @@ export default class ParsedCertificateInfoRowsComponent extends Component {
       };
     });
   }
+
+  get parsingErrors() {
+    if (this.args.model.parsing_errors && this.args.model.parsing_errors.length > 0) {
```

Comment:
can this be `this.args.model.parsing_errors && this.args.model.parsing_errors.length`?

Reply:
yes! it can be